N3.js/src/main/scala/org/w3/banana/n3js/N3.scala

```diff
@@ -0,0 +1,194 @@
+package org.w3.banana
+package n3js
+
+import scala.scalajs.js
+
+object N3 extends js.Object {
+
```

Comment:
Not sure really. N3 is a very specific syntax, that has a few parsers such as [cwm](http://www.w3.org/2000/10/swap/doc/cwm.html) and [euler](http://www.agfa.com/w3c/euler/). This seems to be a more general object that gives you access to a Parser, a Store and some Util libs that can process N3, Ntriples, Turtle, NQuads, ...